libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -339,4 +374,16 @@ void AP_GPS_SBF::broadcast_configuration_failure_reason(void) const
             gcs().send_text(MAV_SEVERITY_INFO, "GPS %d: SBF is not currently logging", state.instance + 1);
         }
     }
+
+    if (gps._auto_config != AP_GPS::GPS_AUTO_CONFIG_DISABLE &&
+        _init_blob_index < ARRAY_SIZE(_initialisation_blob)) {
+        gcs().send_text(MAV_SEVERITY_INFO, "GPS %d: SBF is not fully configured (%d/%d)", state.instance + 1,
+                        _init_blob_index, ARRAY_SIZE(_initialisation_blob));
+    }
+}
+
+bool AP_GPS_SBF::is_configured (void) {
+    return (!gps._raw_data || (RxState & SBF_DISK_ACTIVITY)) &&
+            (gps._auto_config != AP_GPS::GPS_AUTO_CONFIG_DISABLE &&
```

Comment:
This should be `gps._auto_config == AP_GPS::GPS_AUTO_CONFIG_DISABLE ||` right? Otherwise when auto config is disabled this method will always return false.